roles/nuage_master/tasks/firewall.yml

```diff
@@ -0,0 +1,40 @@
+---
+- when: os_firewall_enabled | bool and not os_firewall_use_firewalld | bool
+  block:
+  - name: Add iptables allow rules
+    os_firewall_manage_iptables:
+      name: "{{ item.service }}"
+      action: add
+      protocol: "{{ item.port.split('/')[1] }}"
```

Comment:
@ashcrow, @tbielawa, This syntax is what we currently have and the reason we keep it this way is so that we are compatible with the `firewalld` and our own module `os_firewall_manage_iptables`.

I think the work to refactor that could be done in a separate PR.  I wanted to minimize the change set.  If you feel strongly we can do that work in this PR.

http://docs.ansible.com/ansible/latest/firewalld_module.html